post_office/models.py

```diff
@@ -108,19 +112,25 @@ def prepare_email_message(self):
             html_message = self.html_message
 
         connection = connections[self.backend_alias or 'default']
+        if isinstance(self.headers, dict) or self.expires_at:
+            headers = dict(self.headers or {})
+            if self.expires_at:
+                headers.update({'Expires': self.expires_at.strftime("%a, %-d %b %H:%M:%S %z")})
+        else:
+            headers = None
```

Comment:
Can we simply this to:
```python
if self.expires_at:
    headers = dict(self.headers or {})
    headers.update({'Expires': self.expires_at.strftime("%a, %-d %b %H:%M:%S %z")})
else:
    headers = self.headers
```

Do you think it's better if we add this header during delivery, or whether we should add `Expires` to `self.headers` when email is first created? I have no strong opinion on this, but perhaps adding the header to be stored in `headers` is a bit more explicit.

Reply:
I would add the `Expires` header just before being sent. Reason is, that we use that field to filter our queryset. Remember, emails never are sent after they expired. 
If we put the "expired at" timestamp into the header, we must eventually update it on two different places. If we just use the "Expired" entry in the header, we must filter against a JSON key, which is not very well supported.